quri_sdk_notebooks/examples/0_quri-algo-vm/0_qsci/0_qsci.ipynb

```diff
@@ -1,5 +1,14 @@
 {
   "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "<div class=\"alert alert-info\">\n",
+        "    QSCI is released under a custom source available license. This prohibits redistribution for commercial use. The full license can be found <a href=\"https://github.com/QunaSys/quri-parts-qsci/blob/main/LICENSE\">here</a>.\n",
```

Comment:
The license prohibits the commercial use itself, not only redistribution for it.

```suggestion
        "    QSCI is released under a custom source available license. This prohibits commercial use. The full license can be found <a href=\"https://github.com/QunaSys/quri-parts-qsci/blob/main/LICENSE\">here</a>.\n",
```